### ISSUE TITLE
Did anyone even test this?

### ISSUE THREAD

**Issue by TomasSestak** [bullhorn/taurus#86]

**Comment by nbouvrette** [comment]
I dont think so - this thing doesn't even seem to build for me (like all other Bullhorn packages I tried so far)

<img width="990" alt="image" src="https://user-images.githubusercontent.com/6453322/222214390-224fa946-15cf-4ef3-8980-77576653f6e2.png">
